packages/tokens/properties/font/size.json

```diff
@@ -0,0 +1,66 @@
+{
+  "eds": {
+    "font": {
+      "size": {
+        "xl": {
+          "value": "2rem",
+          "comment": "Font size for Heading 3 sized text"
```

Comment:
Two notes:
- This is skipping Headings 1&2 ([slack thread w/ Sean](https://chanzuckerbergteam.slack.com/archives/CPJN8SDFE/p1605212242002000?thread_ts=1605212052.001800&cid=CPJN8SDFE)) for now
- The namings don't totally match Figma ... the problem is `H4/H5` are marked as `S/XS`, but we also have body text that is Small/Xsmall 🤔 

I think when Sean finalizes guidance in the next week it will clarify both

Reply:
I'm also inclined to name the sizes `h1, h2, ...` so that it's communicated when no element is explicitly provided.